src/main/java/com/werfen/report/test/utils/pdf/PDFComparator.java

```diff
@@ -19,7 +19,7 @@ public static ComparisonResult compareFiles(File expectedFile, File actualFile)
         }
     }
 
-    public static ComparisonResult compareFiles(InputStream expectedStream, InputStream actualStream) throws IOException {
+    public static ComparisonResult compareStreams(InputStream expectedStream, InputStream actualStream) throws IOException {
```

Comment:
I am OK with this rename if this method hasn't been published in a version of the library. Otherwise, this will be a breaking change, so we should upgrade the major version number.

Reply:
Understood, major version number will be updated.